openml-utils/src/test/java/com/feedzai/openml/util/validate/ClassificationValidationUtilsTest.java

```diff
@@ -93,6 +100,36 @@ public void testModelInvalidClassification() {
         )).isInstanceOf(ModelLoadingException.class);
     }
 
+    /**
+     * Tests that a model to load is successfully validated when the parameters are valid.
+     *
+     * @throws ModelLoadingException If there is an error with the validation of the model.
+     * @since @@@feedzai.next.release@@@
+     */
+    @Test
+    public void testValidParamsModelToLoad() throws ModelLoadingException {
```

Comment:
Maybe this is a sign that we should not only check if it doesn't break, but also if the result is as expected. Otherwise, if I open a PR that deletes the method's content, the test will still be valid :P